src/actions/speaker-actions.js

```diff
@@ -786,7 +886,9 @@ export const getSpeakersBySummit =
       expand:
         "accepted_presentations,alternate_presentations,rejected_presentations",
       relations:
-        "accepted_presentations,alternate_presentations,rejected_presentations"
+        "accepted_presentations,alternate_presentations,rejected_presentations",
```

Comment:
i do think that u could restrict more if u do
 relations:
        "accepted_presentations,alternate_presentations,rejected_presentations,accepted_presentations.none,alternate_presentations.none,rejected_presentations.none",